execute/executetest/dependencies.go

```diff
@@ -32,6 +32,7 @@ var testFlags = map[string]interface{}{
 	"optimizeUnionTransformation":      true,
 	"vectorizedMap":                    true,
 	"optimizeAggregateWindow":          true,
+	"optimizeStateTracking":            true,
```

Comment:
Is it desired to leave these toggled on? I just added this for mine, but sort of assumed it shouldn't be committed.

Reply:
I have the same hesitation, but it's what I've been asked to do in the past when adding a new feature flag, and it seems to be the trend.